superset-frontend/cypress-base/cypress/utils/parsePostForm.ts

```diff
@@ -22,7 +22,7 @@
 export default function parsePostForm(requestBody: ArrayBuffer) {
   type ParsedFields = Record<string, string[] | string>;
   if (requestBody.constructor.name !== 'ArrayBuffer') {
-    return (requestBody as unknown) as ParsedFields;
+    return requestBody as unknown as ParsedFields;
```

Comment:
Do we still need the "as unknown" here?

Reply:
it's `prettier` auto-format, we will refactor `as unknown` after monorepo

Reply:
I agree with Evan, let's do a cleanup pass after this to get rid of these unknowns, they greatly aggravate my OCD 😄